.github/workflows/code.yml

```diff
@@ -83,3 +128,25 @@ jobs:
           body: See [Changelog](CHANGELOG.rst) for more details
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  
+  # Container release
+  container-release:
+    needs: [build]
+    runs-on: ubuntu-latest
+    # if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
+    steps:
+      - name: Download artifact of image
+        uses: actions/download-artifact@v2
+        with:
+          name: pandablocks-dev-container
+          path: .
+    
+      - name: Load and push image to ghcr
+        run: |
+          docker load --input pandablocks-dev-container.tar
+          echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u PandABlocks --password-stdin
+          docker tag pandablocks-dev-container ghcr.io/pandablocks/pandablocks-dev-container:latest
+          docker push ghcr.io/pandablocks/pandablocks-dev-container:latest
```

Comment:
We need to support tagging with versions too, have a chat to @gilesknap when he's back

Reply:
@tomtrafford this is my approach to building and pushing tagged container images (this example is for a Python project). 
https://github.com/epics-containers/python3-pip-skeleton/blob/main/.github/workflows/code.yml#L161-L217

Feel free to drop by to discuss.